contents/docs/custom-mutators.mdx

```diff
@@ -1,10 +1,10 @@
 ---
-title: Custom Mutators
+title: Writing Data with Custom Mutators
 ---
 
-_Custom Mutators_ are a new way to write data in Zero that is much more powerful than the original ["CRUD" mutator API](./writing-data).
+Custom Mutators are how data is written in Zero. They are much more powerful than the original ["CRUD" mutator API](./writing-data).
```

Comment:
Instead of comparing to something reader has likely not seen, we should just not compare. Assert that mutators are a very powerful, flexible, elegant way to do mutations.